### PR TITLE
Feature/jcs/api to app

### LINKED ISSUE #17: Top 5 Cities to display upon landing in Home component
- [ ] Home page should render top 5 locations (horizontal scroll) - Hard code in top 5 cities with lat and long.
- [ ] Add initial styling
- [ ] Account for bad data, error handling
- [ ] Test

### PR DESCRIPTION
**Pull request by JoshSevy**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
`Added home render functionality and attached api but began getting api cors fails`
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [x] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`Your message here`
# Message for reviewer:
Still need to add in some more styling and add testing to component render. 
# Checklist:
- [ ] My code has no unused/commented out code
- [ ] My code has no console logs
- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/View/View.scss [hunk 1]
```diff
@@ -1 +1,5 @@
 @import '../styles/_variables.scss';
+
+main {
```
Comment (leighlars) [inline src/View/View.scss:3]: we will have a conflict merge here, but no biggie

File: src/helpers/topCities.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+export const cities = [
```
Comment (ErinUntermeyer) [inline src/helpers/topCities.ts:1]: Love that you broke this out into a helper file.

File: src/Home/Home.tsx [hunk 3]
```diff
@@ -1,11 +1,36 @@
...
-import * as ReactDOM from 'react-dom'
+
+
+import { getLocationData } from '../helpers/apiCalls'
+import { cities } from '../helpers/topCities'
+
 import './Home.scss'
 
 const Home: React.FC = () => {
+	const [topCities, setTopCities] = useState([])
```
Comment (leighlars) [inline src/Home/Home.tsx:10]: Groovy! I dig the icons

File: src/helpers/topCities.ts (hunk 2 above)
Comment (leighlars) [inline src/helpers/topCities.ts:1]: Thanks for the fetch call work!

**Review by leighlars — APPROVED** [review]
Super cool! Excited to merge it all together

**Pull request merged**